app/Models/Entry.php

```diff
@@ -336,10 +337,25 @@ public function link(): string {
 	 * @return string|int
 	 */
 	public function date(bool $raw = false) {
+		$userTimezone = FreshRSS_Context::$user_conf->timezone;
+		$userLanguage = FreshRSS_Context::$user_conf->language;
 		if ($raw) {
 			return $this->date;
+		} else if ($userTimezone == 'Asia/Tehran' || $userTimezone == 'Asia/Kabul') {
```

Comment:
> are all users of this timezone using this calendar?

Indeed, that goes both ways. It could make sense as a default but it's otherwise more or less identical to the font issue.

Reply:
In Iran and Afghanistan it's the official calendar. Also I like to add Hijri calendar system which is common in Arab and Islamic countries. If we add an option (like regional format in windows) for users to select their preferred calendar system it would be a better way I think.

Reply:
Maybe picking another library then? https://packagist.org/?query=Jalali%20Hijri
https://github.com/fisharebest/ext-calendar seems to be quite popular and well maintained